discovery-consul/src/main/resources/reference.conf

```
@@ -13,13 +13,9 @@ akka.discovery {
     consul-host = "127.0.0.1"
     consul-port = 8500
 
-    # Prefix for consul tag with the name of the actor system / application name,
-    # services with this tag present will be found by the discovery mechanism
-    # i.e. `system:test` will be found in cluster if the cluster system is named `test`
-    application-name-tag-prefix = "system:"
-
-    # Prefix for tag containing port number where akka management is set up so that
-    # the seed nodes can be found, an example value for the tag would be `akka-management-port:19999`
-    application-akka-management-port-tag-prefix = "akka-management-port:"
+    # the suffix to discover management service
+    # your code should have logic to register SERVICE_NAME-akka-mangement to consul
+    management-service-suffix = "-akka-management"
```

The old implementation had the same issue that it assumes that it is discovering the akka managementport, where as it could be service discovery for anything e.g. a gRPC service

---

i am not following. did you suggest we to use another suffix?

---

@chbatey we are using this piece of code in our production by a separated file. And as for now, it has been over a year.

For this line, are you suggestion it's better to change it like this:

```suggestion
    service-suffix = "-akka-management"
```

or

```suggestion
    management-service-suffix = "-some-other-suffix"
```

?